code/game/turfs/simulated/wall/misc_walls.dm

```diff
@@ -68,3 +68,7 @@
 	desc = "A huge chunk of rusted reinforced metal."
 	icon = 'icons/turf/walls/rusty_reinforced_wall.dmi'
 	hardness = 15
+
+/turf/closed/wall/shuttle
```

Comment:
Should not actually exist, remove properly instead.

Reply:
fuck

Reply:
yes it shouldn't exist, get it out